hw/ip/sysrst_ctrl/data/sysrst_ctrl_testplan.hjson

```diff
@@ -0,0 +1,200 @@
+            '''
+      milestone: V2
+      tests: ["sysrst_ctrl_pin_access_test"]
+    }
+
+    {
+      name: ec_power_on_reset
+      desc: '''
+            * Verify the EC and power on reset. 
+            * Enable the override value by writing to PIN_ALLOWED_CTL.EC_RST_L_1 and PIN_OUT_CTL.EC_RST_L register. 
+            * Make sure ec_rst_out_l is asserted even after opentitan reset is released.
+            * Set PIN_OUT_CTL.EC_RST_L to 0 to release the ec_rst_out_l reset.
+            '''
+      milestone: V2
+      tests: ["sysrst_ctrl_ec_pwr_on_rst"]
+    }
+
+    {
+      name: flash_write_protect_output
+      desc: '''
+            * Verify the flash write protect.
+            * Make sure flash_wp_out_l signal is asserted low by reading PIN_OUT_CTL.flash_wp_l.
+            * Enable the override function by setting PIN_OUT_CTL.FLASH_WP_L to value 1. 
+            * Check if flash_wp_out_l is released. 
+            '''
+      milestone: V2
+      tests: ["sysrst_ctrl_flash_wr_prot_out"]
+    }
+
+    {
+      name: ultra_low_power_test
+      desc: '''
+            * Verify the ultra low power feature. 
+            * Trigger the ac_present_i, lid_open_in, ec_rst_l_i input keys.                                                                                                                                                                              * Configure ULP_AC_DEBOUNCE_CTL, ULP_LID_DEBOUNCE_CTL and ULP_PWRB_DEBOUNCE_CTL register to set the debounce timer value.                                                                                  
+            * Read the WKUP_STATUS register to check if the event has occured.                                                                                      
+            * Read the ULP_STATUS register and check if the ultra low power wakeup event is detected.
+            '''
+      milestone: V2
+      tests: ["sysrst_ctrl_ultra_low_pwr"]
+    }
+
+    {
+      name: stress_all_with_rand_reset
+      desc: '''
+            * This test runs 3 parallel threads - stress_all, tl_errors and random reset.
+            * After reset is asserted, the test will read and check all valid CSR registers.
+            '''
+      milestone: V3
+      tests: ["sysrst_ctrl_stress_all_with_rand_reset"]
+    }
+  ]
+
+  covergroups: [
+    {
+      name: sysrst_ctrl_ec_rst_pulse_width_cg
+      desc: '''
+        Cover the ec_rst_out_l pulse width range from 10-200ms.
+      '''
+    }
+    {
+      name: sysrst_ctrl_key_intr_cg
+      desc: '''
+        Cover the FSM for input key pressed and released. 
+      '''
+    }
+    {
+      name: sysrst_ctrl_timerfsm_cg
+      desc: '''
+        Cover the timer based FSM model for different debounce time. 
+      '''
+    }
+    {
+      name: sysrst_ctrl_ulpfsm_cg
+      desc: '''
+        Cover the ultra low power fsm model.
+      '''
+    }
+    {
+      name: sysrst_ctrl_combo_detect_action
```

Comment:
Ideally, these should have the same name as the covergroup. In future, we will map these entries to covergroups extracted from the tool coverage report.